.github/workflows/ci_cd_main.yml

```
@@ -12,16 +12,15 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-permissions:
-  contents: write
+permissions: {}
 
 jobs:
 
   doc-build:
     name: "Doc build"
     runs-on: ubuntu-latest
     steps:
-      - uses: ansys/actions/doc-build@main
+      - uses: ansys/actions/doc-build@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/doc-build@main
```
Needs to be reverted before merging